x-pack/plugins/security_solution/cypress/helpers/common.ts

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+/**
+ * Helper function to generate selector by data-test-subj
+ * @param dataTestSubjectValue the value passed to the data-test-subj property of the DOM element
+ */
+export const getDataTestSubjectSelector = (dataTestSubjectValue: string) =>
```

Comment:
Thanks for adding! It would be cool in the future (not for this PR, but for one that revamps our cypress tests) where we add custom commands for our test suite: a la:

https://docs.cypress.io/guides/references/best-practices#Real-World-Example